django_quickbooks/views/service.py

```diff
@@ -166,6 +166,9 @@ def receiveResponseXML(ctx, ticket, response, hresult, message):
         if hresult:
             print("hresult=" + hresult)
             print("message=" + message)
+        # FIXME: I know this is a hacky way to get around encoding, needs a better solution
```

Comment:
Does the response come explicitly in **windows-1252** encoding?

Reply:
yes, please take a look at `data/pos/item_inventory_add_response.xml`